### PR TITLE
my side of the story

### PR DESCRIPTION
**Pull request by TomMCauser**
slide 74 task - learning how to submit a pull request

### CODE REVIEW

**Comment by floridop** [comment]
Hi @TomMCauser  ! Thanks for your submission.

Unfortunately it seems that there were changes in the master that conflict with yours.
Could you perform a merge as described in Tutorial 5 slides 84-96 and integrate those changes?
We do not want to lose any of them, so you can keep what others have written and add yours.
You're free to reorganize the content so that it still makes a story, fix typos and bad English, but please do not modify
the content/meaning of what others have written.
You have time until the 29th of October at 15:00. If you can't make it by that time please let me know.

**Comment by TomMCauser** [comment]
Pulled, resolved and pushed back. Should be good now. Let me know if not.

File: fairytale.md [hunk 1]
```diff
@@ -5,8 +5,14 @@
 In a faraway future, in a distant land  
 lives the *Squonk*
 
+The average Squonk is 7 meters tall and screams loud.
```
[the remark this answers is not in the document]
Comment (TomMCauser) [reply fairytale.md:8]: Hi,

I've got my electromagnetism exam tomorrow, so I'm focusing on that at the time being, however I can fix and resubmit this after my exam tomorrow (I will be home at 8pm-ish I think, exam ends at 7pm).

I'll add it to my calendar to do then if that's okay. (It's after the deadline, so not sure if that's too late)

[the remark this answers is not in the document]
Comment (floridop) [reply fairytale.md:8]: Hi,

Thanks for the update! No need to do anything, concentrate on the physics. More details on canvas.

**Comment by floridop** [comment]
Info updated on canvas

**Pull request merged**